### DISCUSSION TITLE
Questions I have for some reason

### DISCUSSION THREAD

**Discussion by BumbleFucquer** [FunkinCrew/Funkin#1554] [Q&A]
Couple Questions for Devs
1. If Girlfriend's parents are married according to the wiki, why don't they wear wedding rings?
2. Will you ever do any of the unreached kickstarter goals (Pico Remake, Nightmare AU) in the future even though they were unreached?

Sincerely,
BumbleFucquer

P.S. I dare you to try and say my name correctly

**Comment by Windows1Door2Daniel** [comment]
who said is obligatory

**Comment by ghost** [comment]
Note: I don't think married always wear rings.

**Comment by steviegt6** [comment]
> Note: I don't think married always wear rings.

According to all people under the age of 10, married couples must always wear wings, hold hands, and kiss. Known fact.
